comp/core/workloadmeta/collectors/catalog-core/options.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// TODO: (components) Move remote-only to its own catalog, similar to how catalog-less works
+// Depend on this catalog-remote using fx, instead of build tags
+
```

Comment:
nit: 

Would be better to put this comment somewhere in the code below so it is more visible.

People usually skip the top level comments as they are copyrights and package description.

Reply:
I think its probably better to leave it there as it pertains to the build tag right below it.

Reply:
Tag was removed